napari/_tests/test_viewer.py

```diff
@@ -357,3 +357,16 @@ def test_current_viewer(make_napari_viewer):
 
     assert current_viewer() is viewer1
     assert current_viewer() is not viewer2
+
+
+def test_reset_empty(make_napari_viewer):
+    """Test that resetting an empty viewer doesn't crash"""
```

Comment:
Optional: as this test doesn't assert anything and only checks that there is no error, a link to the original issue would be useful for context. I guess we can assume people can find it via blame, so if you think this isn't important, feel free to ignore.